rich/logging.py

```diff
@@ -76,6 +92,16 @@ def emit(self, record: LogRecord) -> None:
         level = Text()
         level.append(record.levelname, log_style)
 
+        traceback = None
+        if self.handle_tracebacks and record.exc_info:
```

Comment:
The Traceback will need to be constructed with the exception in `record.exc_info`, since it's possible to log an exception that wasn't necessarily the most recent one. See [Traceback.from_exception](https://rich.readthedocs.io/en/latest/reference/traceback.html#rich.traceback.Traceback.from_exception)